src/main/java/io/github/debuggyteam/architecture_extensions/RenameThisLater.java

```diff
@@ -0,0 +1,9 @@
+package io.github.debuggyteam.architecture_extensions;
+
+import io.github.debuggyteam.architecture_extensions.api.BlockGroup;
+import io.github.debuggyteam.architecture_extensions.api.BlockType;
+import net.minecraft.block.Block;
+
+public record RenameThisLater(BlockType type, BlockGroup group, Block block) {
```

Comment:
Are there any usages of this record at the moment?

If there are not, we should get rid of it. If there are, we should rename it.

Reply:
This was made when I tried (and failed) to get transparency working for the transoms.